sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/BackupRestoreRestClient.cs

```diff
@@ -49,10 +49,10 @@ internal HttpMessage CreateFullBackupRequest(string vaultBaseUrl, SASTokenParame
             uri.AppendPath("/backup", false);
             uri.AppendQuery("api-version", apiVersion, true);
             request.Uri = uri;
-            request.Headers.Add("Content-Type", "application/json");
             request.Headers.Add("Accept", "application/json");
             if (azureStorageBlobContainerUri != null)
             {
+                request.Headers.Add("Content-Type", "application/json");
```

Comment:
This is not a compatible change. It should be outside the `if`.

Reply:
>...or is the idea that you only set the request Content-Type if there would be a response body?

We only set the content type if there is a request body.

Reply:
The empty body is not a valid `application/json` value.